### DISCUSSION TITLE
Rdio Scanner not stopping when imcoming call on android

### DISCUSSION THREAD

**Discussion by Nirbles** [chuot/rdio-scanner#289] [Q&A]
The Rdio Scanner does not stop when I receive a call on my Android phone. I can't figure out what setting needs to be changed.
Thank you in advance

**Comment by chuot** [comment]
What is your phone brand and what is the OS version?

**Reply by Nirbles** [reply]
Samsung s20+ Android 13.0
The app doesn't seem to be showing a version #
Thank you

**Reply by chuot** [reply]
The native app version is at the bottom of the settings screen on the left.

**Reply by Nirbles** [reply]
Sorry... 1.8.3

**Reply by chuot** [reply]
Ok, so I will try to reproduce on one of my old phone since my daily driver isn't running android. I know it has been problematic in the past when an incoming call is received, but I'm also figthing other audio apps that stop the audio on the rdio-scanner app. I will see if I can do something about it without disrupting too much the app. It will also takes time before I can find a solution, and then I will have to beta test before releasing to the public, so bear with me.

**Reply by Nirbles** [reply]
Thank you sir. Very much appreciated.

**Reply by TPSSCW** [reply]
Has this issue been resolved? My s22 stops running the rdio app randomly

**Reply by Forts117** [reply]
No, it has not been resolved. I've been using my mobile browser to access rdio which works somewhat better than the app, but it's still problematic too.

**Comment by DrHolbr** [comment]
On my pixel 9 pro I had to give it permission to run in the background, AND
go into my phone's battery settings and disable "adaptive battery" as well.
After turning off adaptive battery, my app seems to stay running in the
background indefinitely

**Reply by Forts117** [reply]
Thanks for sharing... I'm using a Pixel 8 and I'll give that a shot.
